tchannel/tornado/connection.py

```diff
@@ -135,6 +141,10 @@ def __init__(self, connection, tchannel=None, direction=None):
 
         connection.set_close_callback(self._on_close)
 
+    @property
+    def num_out_pendings(self):
```

Comment:
each connection will expose its own one and peers' is the debate below about util method or instance method inside peer.